packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -189,6 +191,26 @@ augmentFieldsDict(Tags, {
       }
     }
   },
+  descriptionHtmlWithAnchors: {
+    resolveAs: {
+      type: "String",
+      resolver: async (tag: DbTag, args: {}, context: ResolverContext): Promise<string> => {
+        const description = tag?.description?.originalContents?.data;
+        if (!description) {
+          return "";
+        }
+
+        const usedAnchors: Record<string, boolean> = {};
+        // @ts-ignore DefinitelyTyped annotation is wrong, and cheerio's own annotations aren't ready yet
+        const $ = cheerio.load(description, null, false);
+        $(".table strong").each((i, elem) => {
```

Comment:
Should this not include h1 etc?

Reply:
I think you are fooled by the fact that all headers are currently bold. They are not enforced to be however.